com.microsoft.mrtk.accessibility/Subsystems/MRTK/MRTKAccessibilitySubsystem.cs

```diff
@@ -40,10 +41,153 @@ class AccessibilityProvider : Provider
+            /// <param name="objectList">The collection which will receive the requested <see cref="GameObject"/>s.</param>
+            /// <remarks>When this method is called, the objectList will be cleared prior to adding the requested <see cref="GameObject"/>s.</remarks>
+            private void AssembleDescribableObjects(
+                ObjectClassification classifications,
+                List<GameObject> objectList)
+            {
+                if ((int)(classifications & ObjectClassification.People) != 0)
+                {
+                    objectList.AddRange(describableObjects[ObjectClassification.People]);
+                }
+                if ((int)(classifications & ObjectClassification.Places) != 0)
+                {
+                    objectList.AddRange(describableObjects[ObjectClassification.Places]);
+                }
+                if ((int)(classifications & ObjectClassification.Things) != 0)
+                {
+                    objectList.AddRange(describableObjects[ObjectClassification.Things]);
+                }
+                if ((int)(classifications & ObjectClassification.UserInterface) != 0)
+                {
+                    objectList.AddRange(describableObjects[ObjectClassification.UserInterface]);
+                }
+                if ((int)(classifications & ObjectClassification.Background) != 0)
+                {
+                    objectList.AddRange(describableObjects[ObjectClassification.Background]);
+                }
+            }
+
+            /// <summary>
+            /// Filters the provided collection of <see cref="GameObject"/>s based on the visibility to the main camera
+            /// and the maximum allowed distance.
+            /// </summary>
+            /// <param name="readerView">The desired visibility (in field of view, full surround, etc.) of the <see cref="GameObject"/>s.</param>
+            /// <param name="maxDistance">The distance, in meters, beyond which <see cref="GameObject"/>s will be filtered.</param>
+            /// <param name="objectList">The collection of <see cref="GameObject"/>s that will be filtered based on visibility and distance.</param>
+            private void FilterDescribableObjects(
+                ReaderView readerView,
+                float maxDistance,
+                List<GameObject> objectList)
+            {
+                // Ensure there is a collection to filter.
+                if (objectList.Count == 0) { return; }
+
+                // Walk the list backwards, so that it does not hit a collection changed exception.
+                for (int i = (objectList.Count - 1); i != 0; i--)
+                {
+                    GameObject obj = objectList[i];
+
+                    // Does the object have a collider?
+                    Collider collider = obj.GetComponent<Collider>();
```

Comment:
Do we document or enforce this requirement anywhere? Also, why is it a requirement? Just for `CameraFOVChecker`?

Reply:
currently, it is a requirement of CameraFOVChecker, yes.